meerk40t/core/node/elem_text.py

```diff
@@ -146,7 +157,29 @@ def __copy__(self):
         nd["stroke"] = copy(self.stroke)
         nd["stroke_width"] = copy(self.stroke_width)
         nd["fill"] = copy(self.fill)
-        return TextNode(**nd)
+        newnode = TextNode(**nd)
+        newnode._generator = self._generator
+        newnode.__magnification = self.__magnification
+        newnode._image = self._image
+        return newnode
+
+    @property
+    def _magnification(self):
+        return self.__magnification
+
+    @_magnification.setter
+    def _magnification(self, value):
+        if self.__magnification != value:
+            self.__magnification = value
+            self._image = None
+            self._processed_image = None
+            if self._generator is not None:
+                self._generator(self)
+
+
+    @property
+    def _dpi(self):
+        return 72 * self._magnification
```

Comment:
Why 72? We usually use 96. I assume this is like PPI value? Oh, maybe it's 96 / (4/3) converting pt to px with a 96 ppi.